cmd/fleet/serve.go

```diff
@@ -731,6 +719,21 @@ func cronVulnerabilities(
 	}
 }
 
+func checkVulnerabilities(ctx context.Context, ds fleet.Datastore, logger kitlog.Logger, vulnPath string, config config.FleetConfig) {
+	err := vulnerabilities.TranslateSoftwareToCPE(ctx, ds, vulnPath, logger, config)
+	if err != nil {
+		level.Error(logger).Log("msg", "analyzing vulnerable software: Software->CPE", "err", err)
+		sentry.CaptureException(err)
+		return
+	}
+
+	err = vulnerabilities.TranslateCPEToCVE(ctx, ds, vulnPath, logger, config)
+	if err != nil {
+		level.Error(logger).Log("msg", "analyzing vulnerable software: CPE->CVE", "err", err)
+		sentry.CaptureException(err)
```

Comment:
Nit: I'd leave the `return` (if lint doesn't complain). Future developer adding code below could forget.